api/routers/edr.py

```diff
@@ -105,15 +112,21 @@ async def get_data_position(
 )
 async def get_data_area(
     coords: str = Query(..., example="POLYGON((5.0 52.0, 6.0 52.0,6.0 52.1,5.0 52.1, 5.0 52.0))"),
-    parameter_name: str = Query(..., alias="parameter-name", example="dd,ff,rh,pp,tn"),
-    datetime: str | None = None,
+    parameter_name: str = Query(None, alias="parameter-name", example="dd,ff,rh,pp,tn"),
+    datetime: str = Query(None, example="2022-12-31T00:00Z/2023-01-01T00:00Z"),
     f: str = Query(default="covjson", alias="f", description="Specify return format."),
 ):
-    poly = wkt.loads(coords)
-    assert poly.geom_type == "Polygon"
+    try:
+        poly = wkt.loads(coords)
+        assert poly.geom_type == "Polygon"
```

Comment:
I think we should raise different errors for failure to parse the wkt or if the wrong type of geometry is provided.

Reply:
Added explicit error messages to parsing & wrong geom types.